### ISSUE TITLE
Docs for the config file - dependecies version

### ISSUE THREAD

**Issue by crtdaniele** [dependabot/dependabot-core#2575]
Hi all,
I think is not clear the docs for the dependabot-core use.
In the official site a I finded this guide: https://dependabot.com/docs/config-file/

If a put a config.yml in the folder .dependabot, work correctly.
But is not clear... 

For example, if I use this config:

    ignored_updates:
      - match:
          dependency_name: "@testing-library/jest-dom"

The dependabot work correcly, because in the pipeline I see this: @testing-library/jest-dom (4.2.4) => Dependency ignored

Now, if i want to lock and verify if jest-dom is set correctly in the package.json?

For example I want to ignore the updates and i want to fix the version (4.2.4). How to check if jest-dom is in this version?

Another question that is not clear (I don't find any docs), if I want to configure dependabot-core with a npm private registry? Do you have a documentation?

Thanks for all.
Daniele

**Comment by feelepxyz** [comment]
> Now, if i want to lock and verify if jest-dom is set correctly in the package.json?
> For example I want to ignore the updates and i want to fix the version (4.2.4). How to check if jest-dom is in this version?

I'm not sure I follow, do you want to ignore a specific range of jest-dom and create updates if it goes past a certain range?

You can currently only ignore some versions from updates completely, so you could ignore all versions up to 4.2.4 like so:

```
ignored_updates:
  - match:
      dependency_name: "@testing-library/jest-dom"
      version_requirement: "< 4.2.5"
```

> configure dependabot-core with a npm private registry

We currently don't have docs on this but are working on integrating this functonality in the new github native version of dependabot and launching this in the coming month.

You can configure this in dependabot preview by going to the [dashboard](https://app.dependabot.com/) clicking on your account name in the top right > config variables and adding your private registry host and token. This credential should then get used during the update if a matching registry is used in your project.

**Comment by mwaddell** [comment]
I believe this is addressed in https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates and can be closed
